runbooks/OrphanedVirtualMachineImages.md

```diff
@@ -0,0 +1,77 @@
+# OrphanedVirtualMachineImages
+
+## Meaning
+
+This alert fires when a VMI (virt-launcher-* pod) is running on a node that does not have a running `virt-handler-*` pod.  
+
+## Impact
+
+When a(n) VMI(s) is running on a node that does not have a running `virt-handler-*` those VMIs are considered orphaned. Meaning that they are no longer able to be managed.
+
+## Diagnosis
+
+You can confirm the alert by finding which nodes your virt-hanlder pods are running on.
+
+```bash
+kubectl -n kubevirt get pods -o wide -l kubevirt.io=virt-handler
```

Comment:
kubevirt is not necessarily running on `kubevirt` namespace, could be `kubevirt-hyperconverged` if running with HCO.
maybe
```
kubectl get pods --all-namespaces -o wide -l kubevirt.io=virt-handler
```